openpype/lib/applications.py

```diff
@@ -1044,7 +1044,8 @@ def discover_launch_hooks(self, force=False):
                     hook = klass(self)
                     if not hook.is_valid:
                         self.log.debug(
-                            "Hook is not valid for current launch context."
+                            "Hook is not valid for current "
+                            "launch context: {}".format(str(hook))
```

Comment:
I would change the `str(hook)` to `klass.__name__` to avoid long names in logs:
- e.g. `<pre_copy_template_workfile.CopyTemplateWorkfile object at 0x000001D52BD48388>`

Reply:
Agreed. Will also change the lines below to match that.